### PR TITLE
add Error reporting to gqlopencensus

### PR DESCRIPTION
**Pull request by vvakame**
with https://github.com/99designs/gqlgen/pull/419

datadog-example

### CODE REVIEW

File: gqlopencensus/tracer.go [hunk 1]
```diff
@@ -104,6 +104,28 @@ func (tracerImpl) StartFieldChildExecution(ctx context.Context) context.Context
...
+	if !span.IsRecordingEvents() {
+		return
+	}
+
+	rc := graphql.GetResolverContext(ctx)
+	reqCtx := graphql.GetRequestContext(ctx)
+	errList := reqCtx.GetErrors(rc)
+	if len(errList) != 0 {
+		span.SetStatus(trace.Status{
+			Code:    2, // UNKNOWN, HTTP Mapping: 500 Internal Server Error
```
Comment (vvakame) [inline gqlopencensus/tracer.go]: https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto

**Comment by vektah** [comment]
Is there no enumeration generated that can be used here?

**Comment by vvakame** [comment]
https://godoc.org/google.golang.org/genproto/googleapis/rpc/code#Code is it.
Do we add this dependency and use this?

**Comment by vvakame** [comment]
99designs/gqlgen#419 is merged!

**Pull request merged**